streams/src/test/java/org/apache/kafka/streams/processor/internals/PartitionGroupTest.java

```diff
@@ -117,7 +117,7 @@ public void testTimeTracking() {
         record = group.nextRecord(info);
         // 1:[3, 5]
         // 2:[2, 4, 6]
-        // st: 2
+        // st: 1
```

Comment:
I noticed the comments throughout this test were not accurate, so I fixed them to reflect the current streamtime logic